spec/unit/format_routing_constraint_spec.rb

```diff
@@ -1,56 +1,58 @@
-require "test_helper"
+RSpec.describe FormatRoutingConstraint, type: :model do
+  include ContentStoreHelpers
 
-class FormatRoutingConstraintTest < ActiveSupport::TestCase
-  context "#matches?" do
+  describe "#matches?" do
     context "when the content_store returns a document" do
-      setup do
+      before do
         @format = "foo"
         stub_content_store_has_item("/#{slug}", schema_name: @format)
         @request = request
       end
 
-      should "return true if format matches" do
-        assert subject(@format).matches?(@request)
+      it "returns true if format matches" do
+        expect(subject(@format).matches?(@request)).to be true
```

Comment:
Should probably use `described_class` here rather than `subject`.

Reply:
Have reworked this file to use described_class and let a bit more